### PR TITLE
Updated NCache professional  5.3.2 docker files amd scripts.

### PR DESCRIPTION
**Pull request by zeeshanmjd**

### CODE REVIEW

File: professional/client/WindowServer/Dockerfile [hunk 1]
```diff
@@ -15,12 +17,12 @@ COPY resources .
...
 # For more information about ports visit https://www.alachisoft.com/resources/docs/ncache/getting-started-guide-windows/open-firewall-tcp.html;
-EXPOSE 8250-8260 9800 7800-7900 8300-8400
+EXPOSE 8250-8260 9800 8300-8400
 
-# Installing the .NET 6.0 runtime.
-# .NET 6.0 can be downloaded from https://dotnet.microsoft.com/en-us/download/dotnet.
-RUN "powershell.exe Start-Process -Filepath <Filepath of .NET 6.0 from resources folder> -NoNewWindow -Wait -PassThru -ArgumentList '/s'; \
-Remove-Item <Filepath to .NET 6.0 in working directory>;"
+#downloading and installing the .NET 6.0.
+RUN curl -SL --output dotnet-runtime-win-x64.exe https://dotnetcli.azureedge.net/dotnet/Runtime/%DOTNET_VERSION%/dotnet-runtime-%DOTNET_VERSION%-win-x64.exe \
```
Comment (asyncpoint) [inline professional/client/WindowServer/Dockerfile]: Rename and use the downloaded file as 'dotnet.exe' to make it edition natural

File: professional/server/WindowServer/Dockerfile [hunk 2]
```diff
@@ -17,10 +19,10 @@ COPY resources .
 # For more information about ports visit https://www.alachisoft.com/resources/docs/ncache/getting-started-guide-windows/open-firewall-tcp.html;
 EXPOSE 8250-8260 9800 7800-7900 8300-8400
 
-# Installing the .NET 6.0 runtime.
-# .NET 6.0 can be downloaded from https://dotnet.microsoft.com/en-us/download/dotnet.
-RUN "powershell.exe Start-Process -Filepath <Filepath of .NET 6.0 from resources folder> -NoNewWindow -Wait -PassThru -ArgumentList '/s'; \
-Remove-Item <Filepath to .NET 6.0 in working directory>;"
+#downloading and installing the .NET 6.0.
+RUN curl -SL --output dotnet-runtime-win-x64.exe https://dotnetcli.azureedge.net/dotnet/Runtime/%DOTNET_VERSION%/dotnet-runtime-%DOTNET_VERSION%-win-x64.exe \
```
Comment (asyncpoint) [inline professional/server/WindowServer/Dockerfile]: Make it dotnet.exe, to avoid future renaming.

**Review by asyncpoint — CHANGES_REQUESTED** [review]

File: professional/client/WindowServer/Dockerfile (hunk 1 above)
Comment (zeeshanmjd) [reply professional/client/WindowServer/Dockerfile]: updated

File: professional/server/WindowServer/Dockerfile (hunk 2 above)
Comment (zeeshanmjd) [reply professional/server/WindowServer/Dockerfile]: updated

**Pull request merged**